### ISSUE TITLE
Native compilation for armhf fails

### ISSUE THREAD

**Issue by bmhm** [java-native/java-simple-serial-connector#24]
if you execute
`mvn clean install`

on an raspberry pi, the compilation fails.
Reason: The `libjssc.so` will be put into the `linux_32` directory.

It should be in `linux_arm` instead.

Thats a problem of the current `native-lib-loader` convention. The new one will fix this issue automagically.

Workaround: `-DskipTests=true`

**Comment by tresf** [comment]
This seems like it's better tracked at `native-lib-loader` repo.  Since there's no work to be done here.  Thanks for documenting.

**Comment by tresf** [comment]
... on second thought... can you try adding these profiles @bmhm?

```xml
    <!-- fix arm output directories -->
    <!-- TODO: See issue #24; remove w/ newer native-lib-loader -->
    <profile>
      <id>arm-fix32</id>
      <activation>
        <os>
          <arch>arm</arch>
        </os>
      </activation>
      <properties>
        <cmake.generate.extra>-DNATIVE_LIB_DIR=${os.family}_${os.arch}</cmake.generate.extra>
      </properties>
    </profile>
    <profile>
      <id>arm-fix64</id>
      <activation>
        <os>
          <arch>aarch_64</arch>
        </os>
      </activation>
      <properties>
        <cmake.generate.extra>-DNATIVE_LIB_DIR=${os.family}_arm64</cmake.generate.extra>
      </properties>
    </profile>
```

**Comment by tresf** [comment]
@bmhm has confirmed this is fixed by 88e7954.  Closing as the bug report is in regards to an open PR.
